cogstat/cogstat_chart.py

```diff
@@ -725,6 +725,107 @@ def create_variable_pair_chart(data, meas_lev, x, y, result=None, raw_data=False
         graph = plt.gcf()
     return graph
 
+def create_multiple_variable_chart(data, meas_lev, x, type="raw"):
+    """Draw a chart relating more than two variables displaying raw data
+
+    Parameters
+    ----------
+    data : pandas dataframe
+    meas_lev : {'int', 'ord', 'nom', 'unk'}
+        Measurement level of the variables
+    x : str
+        Name of the x variable.
+    type : {'raw', 'collinearity'}
+        Displays raw data when 'raw', displays scatterplot matrix of explanatory variables when 'collinearity'.
+
+    Returns
+    -------
+    matplotlib chart
+        A matrix plot of the variables optionally containing the raw data.
+    """
+
+    if meas_lev == "int":
+        if type == "collinearity":
+            data = data[x]
+        if type in "raw":
+            fig, ax = plt.subplots(len(data.columns), len(data.columns), tight_layout=True)
+            fig.suptitle(_plt("Scatterplot matrix of variables"))
+            for i in range(0, len(data.columns)):
+                ax[i, 0].set_ylabel(data.columns[i])
+                ax[len(data.columns) - 1, i].set_xlabel(data.columns[i])
+                for j in range(0, len(data.columns)):
+                    if i == j:
+                        ax[i, j].hist(data.iloc[:, i])
+                    else:
+                        ax[i, j].scatter(data.iloc[:, i], data.iloc[:, j])
+        elif type is "collinearity":
+            if len(x) == 2:
+                ncols = 1
+            elif len(x) >= 2:
+                ncols = 2
+            import math
+            nrows = math.ceil(len(x)/2)
+            fig = plt.figure(tight_layout=True)
+            fig.suptitle(_plt("Scatterplot matrix of explanatory variables"))
+            x_done = []
+            ind = 1
+            for index_1, x_i in enumerate(x):
```

Comment:
then x is not str but a list of str? plase, modify the docstring (and rename the x related variables here too)

Reply:
"x" renamed, docstring updated.